analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/SymbolReferenceAnalyzerTest.cs

```diff
@@ -115,25 +115,12 @@ public class SymbolReferenceAnalyzerTest
         public void Verify_Property_CS(ProjectType projectType) =>
             Verify("Property.cs", projectType, 4, 3, 7, 8);
 
-        [DataTestMethod]
-        [DataRow(ProjectType.Product)]
-        [DataRow(ProjectType.Test)]
-        public void Verify_Setter_CS(ProjectType projectType) =>
-            Verify("Setter.cs", projectType, 4, 6, 8);
-
         [DataTestMethod]
         [DataRow(ProjectType.Product)]
         [DataRow(ProjectType.Test)]
         public void Verify_TypeParameter_CS(ProjectType projectType) =>
             Verify("TypeParameter.cs", projectType, 5, 2, 4, 6);
 
-        [DataTestMethod]
-        [DataRow(true)]
-        [DataRow(false)]
-        public void GetSetKeyword_ReturnsNull_VB(bool isTestProject) =>
```

Comment:
"set" keyword was supported only for adding support for "value". This complicated the logic and it was not consistent. The "set" declaration was added only in the presence of "value".

I prefer to remove this part of the functionality and keep the logic simple.